src/test/unit/rx_crsf_unittest.cc

```diff
@@ -349,16 +362,18 @@ TEST(CrossFireTest, TestCrsfDataReceive)
 
 extern "C" {
 
-int16_t debug[DEBUG16_VALUE_COUNT];
-uint32_t micros(void) {return dummyTimeUs;}
-uint32_t microsISR(void) {return micros();}
-serialPort_t *openSerialPort(serialPortIdentifier_e, serialPortFunction_e, serialReceiveCallbackPtr, void *, uint32_t, portMode_e, portOptions_e) {return NULL;}
-const serialPortConfig_t *findSerialPortConfig(serialPortFunction_e ) {return NULL;}
-bool telemetryCheckRxPortShared(const serialPortConfig_t *) {return false;}
-serialPort_t *telemetrySharedPort = NULL;
-void crsfScheduleDeviceInfoResponse(void) {};
-void crsfScheduleMspResponse(void) {};
-bool bufferMspFrame(uint8_t *, int) {return true;}
-bool isBatteryVoltageAvailable(void) { return true; }
-bool isAmperageAvailable(void) { return true; }
+    int16_t debug[DEBUG16_VALUE_COUNT];
+    uint32_t micros(void) {return dummyTimeUs;}
```

Comment:
```suggestion
    uint32_t micros(void) { return dummyTimeUs; }
```
This is optional and how I prefer, not sure what our code guide is saying, please correct the ones below to be consistent please.